packages/tasks/src/model-libraries.ts

```diff
@@ -192,6 +192,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.mlx,
 		filter: true,
 	},
+	"mlxim": {
```

Comment:
```suggestion
	mlxim: {
```

Linter will complain otherwise :)